symbolic-symcache/src/writer.rs

```diff
@@ -298,12 +299,13 @@ where
             s => s,
         };
 
+        let len = NonZeroU16::new(std::cmp::min(size, 0xffff) as u16)
+            .expect("Function length must be positive");
```

Comment:
This can panic, which we would avoid ideally. Given that this only happens if we have a bug somewhere, could we instead only panic in debug and otherwise gracefully skip?

Reply:
I just realized that due to the lines directly before that, `size` can never be 0 anyway.

Reply:
Of course, you're right.